frontend/app/views/spree/products/index.html.erb

```diff
@@ -1,7 +1,9 @@
 <% content_for :sidebar do %>
   <div data-hook="homepage_sidebar_navigation">
     <% if "spree/products" == params[:controller] && @taxon %>
-      <%= render :partial => 'spree/shared/filters' %>
+      <% Spree::Config.search_filter_presenter_class.new(taxon: @taxon).tap do |presenter| %>
```

Comment:
I'm starting to wander if it's worth moving this inside the controller and saved in an instance variable.